Fix doc build with Sphinx 8.

Romain-Geissler-1A opened the pull request:
See this Fedora failure following the upgrade to Sphinx 8: https://bugzilla.redhat.com/show_bug.cgi?id=2329853

Romain-Geissler-1A committed:
Fix doc build with Sphinx 8.

See this Fedora failure following the upgrade to Sphinx 8:
https://bugzilla.redhat.com/show_bug.cgi?id=2329853

twm committed:
Merge branch 'trunk' into fix-build-sphinx-8

twm committed:
Add Sphinx build to CI

twm commented:
The change here looks good, however we don't actually seem to do a Sphinx build in CI so I dropped in a step to to that. As a consequence I can't approve.

twm committed:
Sphinx: Add language config

Addressing this warning:

    WARNING: Invalid configuration value found: 'language = None'. Update your configuration to a valid language code. Falling back to 'en' (English).

glyph commented:
life is a highway
